molgenis-data-import/src/main/java/org/molgenis/data/importer/ImportWriter.java

```diff
@@ -78,6 +78,9 @@ public ImportWriter(DataService dataService, PermissionSystemService permissionS
 	@Transactional
 	public EntityImportReport doImport(EmxImportJob job)
 	{
+		ImportedDataToBackendCompatibilityValidator validator = new ImportedDataToBackendCompatibilityValidator(dataService);
+		validator.validate(job.parsedMetaData.getEntities());
```

Comment:
Agreed, you need to check entities existing in the database, some of them not owned by you probably